### DISCUSSION TITLE
espnow receiver  display audio start stop indication on lcd

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#854] [Q&A] [answered]
I am sending audio from esp1 to esp2 via espnow  but how to get information at receiver end that audio has completed or not or when the new audio has started .

**Comment by pschatzmann** [comment, marked as answer]
This obviously depends on the application and there is therefore no generic mechanism.
One simple way would be to check if stop receiving data during some short period of time or to send send some metdata that indicates that there is a new title
